index.js

```diff
@@ -125,7 +125,17 @@ async function post ({ archivePath, target, packmgrPath, checkIfUp }) {
 
 async function check (target) {
   try {
-    const res = await fetch(target)
+    let url = target
+    const headers = new Headers()
+    const regex = /\/\/(.*:.*)@/gm
```

Comment:
Use `new URL()` instead of regular expression:
```js
const url = new URL(target)
const auth = `${url.username}:${url.password}`
url.username = ''
url.password = ''
const res = await fetch(url.toString(), {
  headers: { 'Authorization':  `Basic ${btoa(auth) }` }
}
```